src/Tools/dotnet-watch/src/Internal/MsBuildFileSetFactory.cs

```diff
@@ -86,23 +91,42 @@ public async Task<IFileSet> CreateAsync(CancellationToken cancellationToken)
                     if (exitCode == 0 && File.Exists(watchList))
                     {
                         var lines = File.ReadAllLines(watchList);
-                        var isNetCoreApp31OrNewer = lines.FirstOrDefault() == "true";
 
-                        var fileset = new FileSet(
-                            isNetCoreApp31OrNewer,
-                            lines.Skip(1)
-                                .Select(l => l?.Trim())
-                                .Where(l => !string.IsNullOrEmpty(l)));
+                        var staticFiles = new List<string>();
+                        var commandLine = new CommandLineApplication();
```

Comment:
Context for this change? Confused why you are creating a CommandLineApplication here.